docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
+    build: .
```

Comment:
I usely let the docker-compose.yml file in the root folder (with composer.json, travis.yml, bower.json, etc...)